packages/css/list.css

```diff
@@ -1,22 +1,21 @@
 .ds-list {
+  --dsc-list-font-size: var(--ds-font-size-5);
   --dsc-list-padding-left: var(--ds-spacing-6);
+  --dsc-list-spacing: var(--ds-spacing-2);
 
+  font-size: var(--dsc-list-font-size); /* Replace with composes paragraph-md */
+  line-height: var(--ds-line-height-md); /* Replace with composes paragraph-md */
```

Comment:
Quick note, but we are now doing this differently. I think it's best to keep `asChild` on typography elements, and later remove this is favor of composes. 
Here you have forgotten to set letter spacing and font weight.

Reply:
Oh, thought we agreed to rather add a comment in CSS, but I can keep Paragraph in the meantime ☺️